code42_connector.py

```diff
@@ -398,6 +549,24 @@ def _get_user(self, username):
     def _get_user_id(self, username):
         return self._get_user(username)["userUid"]
 
+    def _get_file_content(self, file_hash):
+        if utils.is_md5(file_hash):
+            response = self._client.securitydata.stream_file_by_md5(file_hash)
+        elif utils.is_sha256(file_hash):
+            response = self._client.securitydata.stream_file_by_sha256(file_hash)
+        else:
+            raise Code42UnsupportedHashError()
+
+        chunks = [chunk for chunk in response.iter_content(chunk_size=128) if chunk]
```

Comment:
Is chunking even necessary here since we're not doing anything in between the chunks? We should just be able to return `r.content` straight away, no?

Reply:
I think we have to do this because of this line: https://github.com/code42/py42/blob/master/src/py42/services/storage/preservationdata.py#L50

When `stream=False`, we are able to use the `content` property, otherwise the content is not returned in full.. At least according to https://py42docs.code42.com/en/stable/methoddocs/response.html#py42.response.Py42Response.iter_content

Reply:
Although, now that you bring this up, we could potentially have a very large file in memory at this point. I am not sure if that is avoidable or not, I have no idea how this part of Phantom works yet